app/views/layouts/_user_menu.html.haml

```diff
@@ -21,7 +21,10 @@
     = link_to(destroy_user_session_path, :method=>'delete') do
       %span.fa.fa-minus
       Sign out
-- if can? :manage, Conference
+- if current_user.is_admin || (current_user.has_any_role? :admin, { :name => :organizer, :resource => :any },
```

Comment:
I mean the specific code that allows for the users you mentioned to have access to the conference, and that defines what kind of access they have. Those lines tell me nothing about a user with cfp being able to manage the conference model